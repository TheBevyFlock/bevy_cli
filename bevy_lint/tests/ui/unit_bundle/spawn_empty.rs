#![feature(register_tool)]
#![register_tool(bevy)]
#![deny(bevy::unit_bundle)]

use bevy::prelude::*;

fn main() {
    App::new().add_systems(Startup, my_system);
}

fn my_system(mut commands: Commands) {
    commands.spawn(());
    //~^ ERROR: created a `Bundle` containing a unit `()`
}
