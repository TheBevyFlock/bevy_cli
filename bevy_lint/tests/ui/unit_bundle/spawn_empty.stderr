error: created a `Bundle` containing a unit `()`
  --> tests/ui/unit_bundle/spawn_empty.rs:12:20
   |
12 |     commands.spawn(());
   |                    ^^
   |
   = note: units `()` are not `Component`s and will be skipped
note: the lint level is defined here
  --> tests/ui/unit_bundle/spawn_empty.rs:3:9
   |
3  | #![deny(bevy::unit_bundle)]
   |         ^^^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

